scripts/rel/macos-sign-binaries.sh

```diff
@@ -81,6 +81,7 @@ for f in \
         liberocksdb.so \
         libquicer_nif.so \
         libquicer_nif*.dylib \
+        libmsquic*.dylib \
```

Comment:
I think it shouldn't be needed.

I think it caused by the build which is not downloading the prebuilds?